### DISCUSSION TITLE
Sub Building Need: More Shell Slope Variants

### DISCUSSION THREAD

**Discussion by VaelophisNyx** [FakeFishGames/Barotrauma#12422] [Ideas and suggestions]
So we currently have a gap between 04 and 14° for standard shells. This is quit irritating as a sub builder, as it very heavily limits how I can craft a sub hull to fit a specific shape. Either I have to sacrifice accuracy for optimization, or absolutely thrash performance and balance for accuracy.

I'd propose we need:

2°
6°
8°
10°

as well as all existing angles in the form of a straight slope, no curve
Some may see that as bloat
I see it as potential

**Comment by VaelophisNyx** [comment]
the alternative, much better solution would be to just...let us draw the shell shapes as we wish
but that's a hell of a lot of code that doesn't exist

**Comment by Arctodites** [comment]
It would be great to have more options for large inside wall slopes, too, even if it were just 5, 10, 15, 20, 45, 60 degree pieces.  Likewise, to have more options for stair rise / slope, or to have ramps.
